xpk.py

```diff
@@ -2129,10 +2386,10 @@ def get_main_container(args, system, docker_image, resource_type) -> str:
 
   yaml = """- name: {args.docker_name}
                 image: {docker_image}
-                env: {args.env}
+                imagePullPolicy: Always
```

Comment:
Hmm I am not sure if this setting is benefitical for all TPU cases with non-pathways. I remember it slowed down workload start up at large scale (like 5k vms) workloads. 

We can make this pathways specific for now to avoid changing non-pathways flows for now

Reply:
SG, makes sense. Thanks for flagging that!